vmaas_sync/send_messages.go

```diff
@@ -12,15 +12,19 @@ import (
 const BatchSize = 4000
 
 func sendReevaluationMessages() error {
-	var inventoryIDs []string
-	err := database.Db.Model(&models.SystemPlatform{}).
-		Pluck("inventory_id", &inventoryIDs).Error
+	var getInventorIDsFun func() ([]string, error)
+
+	if enabledRepoBasedReeval {
+		getInventorIDsFun = getCurrentRepoBasedInventoryIDs
+	} else {
+		getInventorIDsFun = getAllInventoryIDs
+	}
+	inventoryIDs, err := getInventorIDsFun()
```

Comment:
You could just declare inventoryIDs & err before and not create func variable here.